datasets/arc/arc.py

```diff
@@ -0,0 +1,132 @@
+"""TODO(arc): Add a description here."""
```

Comment:
Actually, we don't need arc dataset right now while MultiRC is more urgently needed.